content/articles/compiled-redesign.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "CompilED for Inclusion"
+date: 2019-04-11
+type: "post"
+authors: ["mustapha"]
+tags: ["accessibility","inclusive design"]
+lede: "I restructured and redesigned the CTL DevTeam’s website CompilED to meet
+the web accessibility standards, and to improve its overall user experience
+inclusively on all devices."
+shortlede: "This site’s latest iteration has improved design and WCAG 2.0
+compliance."
+poster: "poster-compiled-v3.jpg"
+socmediaimg: ""
+hiliteimg: "poster-compiled-v3.jpg"
+poster_sourceurl: ""
+poster_source: ""
+bookendanimal: "dragon"
+---
+
+In March, I restructured and redesigned the CTL DevTeam’s website _CompilED_ to
+meet the web accessibility standards, and to improve its overall user
+experience inclusively on all devices.
+
+This latest iteration of _CompilED_ follows the
+[Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/)
+level AA, and it is much more technically and usably accessible than the
+previous versions. It is keyboard accessible with improved focus styling. The
+site is also seamlessly adaptive to multi-size screen devices—smartphones,
```

Comment:
Is this dash a dash or an Em dash? Its subtle, but I'd use the em dash after 'devices' to better differentiate from 'multi-size'

Reply:
`multi-` is dash. the other one is em dash. It is proper, and it shows up on the page.